clouds/aws/roles/roles.py

```diff
@@ -3,6 +3,7 @@
 
 LOGGER = get_logger(name=__name__)
 DEFAULT_AWS_REGION = "us-east-1"
+IAM_ROLES_MAX_ITEMS = 1000
```

Comment:
no need to set as global; used in one function. save there and re-use

Reply:
Fixed